src/test/regress/multi_1_schedule

```diff
@@ -312,13 +312,16 @@ test: ssl_by_default
 
 # ---------
 # object distribution tests
+# TODO: After deprecating parameterless create_distributed_function combine
```

Comment:
do we really need this todo? I think it is fine if we keep files separate?

Reply:
I find it useful to not to have separate create_distributed_function and function_propagation test. Let's keep it is at is to make sure that we will check it while deprecating create_distributed_function udf.